src/Sdk/Coverage/Coverage.ts

```diff
@@ -0,0 +1,51 @@
+import { Api } from 'Api';
+import { Coverage } from 'types';
+
+import { Options } from '../types';
+import { coverageUrl } from '../routing';
+import { getItemId } from '../utils';
+
+import { CoverageListResponse } from './types';
+
+export default class Coverages {
```

Comment:
FYI `Coverage` does not have plural form. It's just `Coverage`.

Reply:
Oh, right. But when I was trying to rename it, there is a conflict with the `Coverage` data type. I've picked `CoverageSdk` to avoid this conflict. Or does anyone have a better naming suggestion?